ui/hooks/useIsOriginalNativeTokenSymbol.ts

```diff
@@ -0,0 +1,77 @@
+import { useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+import { CaipChainId, Hex } from '@metamask/utils';
+import { useSafeChainsListValidationSelector } from '../selectors';
+import {
+  getMultichainIsEvm,
+  getMultichainCurrentNetwork,
+} from '../selectors/multichain';
+
+// TODO: Remove restricted import
+// eslint-disable-next-line import/no-restricted-paths
+import { getValidUrl } from '../../app/scripts/lib/util';
+import { isOriginalNativeTokenSymbol } from '../helpers/utils/isOriginalNativeTokenSymbol';
+
+export function useIsOriginalNativeTokenSymbol(
+  chainId: Hex | CaipChainId,
+  ticker: string,
+  type: string,
+  rpcUrl = '',
```

Comment:
nit - these 2 fields are never used now. We could remove them, but would be a large refactor on other files (this PR is already too large)